cookbooks/aws-parallelcluster-install/recipes/nvidia.rb

```diff
@@ -83,6 +83,30 @@
     creates "/usr/local/cuda-#{node['cluster']['nvidia']['cuda_version']}"
   end
 
+  # Get CUDA Sample Files
+  cuda_tmp_samplefile = "/tmp/cuda-sample.tar.gz"
+  remote_file cuda_tmp_samplefile do
+    source node['cluster']['nvidia']['sample_url']
+    mode '0644'
+    retries 3
+    retry_delay 5
+    not_if { ::File.exist?("/usr/local/cuda-#{node['cluster']['nvidia']['cuda_version']}/samples") }
```

Comment:
This path is built twice (here and in line 107). What about having a single declaration and reuse the variable?

Reply:
Fixed